src/extras/TilingSprite.js

```diff
@@ -56,25 +47,78 @@ export default class TilingSprite extends core.Sprite
          */
         this._height = height;
 
+        this._canvasPattern = null;
+
+        this._glDatas = [];
+
         /**
-         * An internal WebGL UV cache.
-         *
-         * @member {PIXI.TextureUvs}
-         * @private
+         * transform that is applied to UV to get the texture coords
+         * @member {PIXI.extras.TextureTransform}
          */
-        this._uvs = new core.TextureUvs();
+        this.uvTransform = texture.transform || new TextureTransform(texture);
```

Comment:
Its a component that can be shared between things. Its up to user whether to pre-create it or not.